pages/api/v1/answers.ts

```diff
@@ -181,6 +182,25 @@ export default async function handler(req: NextRequest) {
           );
         if (insertedChatMessagesRes.error)
           throw new Error(insertedChatMessagesRes.error.message);
+
+        const { data, error } = await supabase
+          .from("organizations")
+          .select("openai_usage")
+          .eq("id", org!.id)
+          .single();
+        if (error) throw new Error(error.message);
+        if (!data.openai_usage) throw new Error("No usage found");
+        const usage = data.openai_usage as { [key: string]: number };
+        const todaysDate = new Date().toISOString().split("T")[0];
+        if (todaysDate in usage) {
+          usage[todaysDate] += cost;
+        } else {
+          usage[todaysDate] = cost;
+        }
+        await supabase
+          .from("organizations")
+          .update({ openai_usage: usage })
+          .eq("id", org!.id);
```

Comment:
Feels to me like either 
a) we use the Openai billing API or
b) there should be a new database table for usage and each OpenAI call should be a row in it. This way you:
1. Get the ability to do more granular breakdowns for free
2. Can track cost immediately after each openai call, so if an error happens, we still track usage